QWC2Components/utils/LayerUtils.js

```diff
@@ -79,7 +79,8 @@ const assign = require('object-assign');
         if(layer.type !== "wms") {
             return "";
         }
-        return layer.url + "?SERVICE=WMS&REQUEST=GetLegendGraphic&VERSION=1.3.0&FORMAT=image/png&LAYER=" + sublayer;
+        layer.version=layer.version?layer.version:"1.3.0";
```

Comment:
Perhaps you could add the "1.3.0" fallback directly in ThemeSwitcher.jsx to avoid the additional conditional here?